### PR TITLE
Update metadata

### PR DESCRIPTION
**Pull request by saberduck**

### CODE REVIEW

File: sonar-plugin/javascript-checks/src/main/resources/org/sonar/l10n/javascript/rules/javascript/S3699.html [hunk 1]
```diff
@@ -22,7 +26,8 @@ <h2>Why is this an issue?</h2>
 <h2>Resources</h2>
 <h3>Documentation</h3>
 <ul>
-  <li> <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/undefined">MDN - <code>undefined</code></a> </li>
-  <li> <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/return">MDN - <code>return</code></a> </li>
+  <li> MDN - <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/undefined">MDN - <code>undefined</code></a>
+  </li>
+  <li> MDN - <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/return">MDN - <code>return</code></a> </li>
```
[the remark this answers is not in the document]
Comment (yassin-kammoun-sonarsource) [reply sonar-plugin/javascript-checks/src/main/resources/org/sonar/l10n/javascript/rules/javascript/S3699.html:31]: There are other occurrences of this.

**Review by yassin-kammoun-sonarsource — APPROVED** [review]
Looks good to me overall, but something is weird with RSPECs that include links to MDN.

File: sonar-plugin/javascript-checks/src/main/resources/org/sonar/l10n/javascript/rules/javascript/S3699.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (saberduck) [reply sonar-plugin/javascript-checks/src/main/resources/org/sonar/l10n/javascript/rules/javascript/S3699.html:31]: good catch, I don't know why this was changed. I asked @maksim-grebeniuk-sonarsource , it was done in this PR https://github.com/SonarSource/rspec/pull/2759/files#diff-e3b92eda1cf477470213b880e36f8ae3d9e707e4f2339c4c53a472376653fe23R37

[the remark this answers is not in the document]
Comment (saberduck) [reply sonar-plugin/javascript-checks/src/main/resources/org/sonar/l10n/javascript/rules/javascript/S3699.html:31]: I created PR to fix this https://github.com/SonarSource/rspec/pull/2932 . I didn't find other occurrences of this problem. @yassin-kammoun-sonarsource could you pls have a look?

**Pull request merged**